src/fexists.ts

```diff
@@ -9,20 +9,24 @@ export function fexists(
 	parentPath: FSPath,
 	specifier: Specifier,
 ) {
-	const parentUrl = `${pathToFileURL(dirname(parentPath)).href}/`;
-	const resolvedSpecifier = URL.canParse(specifier)
-		? specifier
-		// import.meta.resolve here is required because we need node's resolution algorithm to
-		// incorporate the results of any hooks that may be helping, such as ones facilitating
-		// tsconfig.compileOptions.paths
-		: fileURLToPath(import.meta.resolve(specifier, parentUrl));
+	try {
```

Comment:
Why does it need the try/catch? (`access` already handles errors)

Reply:
Ahh, let's wrap only that part then (and include a comment)

Reply:
done